server/graphql/v2/object/MemberInvitation.ts

```diff
@@ -0,0 +1,64 @@
+import { GraphQLInt, GraphQLObjectType, GraphQLString } from 'graphql';
+import { GraphQLDateTime } from 'graphql-iso-date';
+
+import { MemberRole } from '../enum/MemberRole';
+import { Account } from '../interface/Account';
+
+import { Tier } from './Tier';
+
+export const MemberInvitation = new GraphQLObjectType({
+  name: 'MemberInvitation',
+  description: 'An invitation to join the members of a collective',
+  fields: () => {
+    return {
+      id: {
+        type: GraphQLInt,
```

Comment:
`id`s are non-null, and usually encoded with GQLV2's API. See https://github.com/opencollective/opencollective-api/blob/main/server/graphql/v2/object/ConnectedAccount.ts#L15 for example

Reply:
Updated 👍🏻